src/portable/project.rs

```diff
@@ -199,7 +189,9 @@ pub struct Upgrade {
     ])]
     pub to_latest: bool,
 
-    /// Upgrade specified instance to a specified version
+    /// Upgrade specified instance to a specified version.
+    /// 
+    /// e.g. --to-version=4.0-beta.1
```

Comment:
I recommend removing the `=` so that the example matches the option syntax we show, which uses a space instead of `=`.
```suggestion
    /// e.g. --to-version 4.0-beta.1
```

Reply:
Interesting, looks like the = isn't a necessity. Looks better without the = for sure.